PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -3394,6 +3412,41 @@ void FindReplaceDlg::drawItem(LPDRAWITEMSTRUCT lpDrawItemStruct)
 	RECT rect;
 	_statusBar.getClientRect(rect);
 	::DrawText(lpDrawItemStruct->hDC, ptStr, lstrlen(ptStr), &rect, DT_SINGLELINE | DT_VCENTER | DT_LEFT);
+
+	if (_statusbarTooltipMsg.length() == 0) return;
+
+	SIZE size;
+	::GetTextExtentPoint32(lpDrawItemStruct->hDC, ptStr, lstrlen(ptStr), &size);
+	int s = (rect.bottom - rect.top) & 0x70; // limit s to available icon sizes and avoid uneven scalings

```

Comment:
Please turn magic number 0x70 into a symbolic constant, maybe something like ICON_SIZE_MASK.

Reply:
There is no advantage in a one time used constant. Compared to the comment, ICON_SIZE_MASK says nothing, as less as any other name I could invent. It's simply an arithmetic operation, like + 3 or - 5. I wouldn't have the idea to invent constants like FIVE or THREE.

Reply:
That works for a single-use constant.

This is not the place for an extended discussion, however I will say that I would not ask for constants such as "FIVE" or "THREE"; they wouldn't add _meaning_ vs. 5 or 3.

Code is a shared facility, read 10 to 100 times as often as it is written.  The idea is to communicate your _intent_ to the next developer who is reading and trying to understand the code.  

It also helps keep them from doing a global search-and-replace of 3.

Reply:
Yes, this argumentation works for the negative error numbers, but not for the

`size = size & 0x70`

Here, the comment in the line is hint enough.